corehq/motech/repeaters/models.py

```diff
@@ -164,6 +167,44 @@ def log_repeater_success_in_datadog(domain, status_code, repeater_type):
     })
 
 
+class RepeaterStubManager(models.Manager):
+
+    def all_ready(self):
+        """
+        Return all RepeaterStubs ready to be forwarded.
+        """
+        return (
+            self.get_queryset()
+            # Is not paused
```

Comment:
nit: I think this comment is redundant with the code. I think you wanted to make this code more readable, you could add in some named variables instead

Reply:
Ooo! I like that idea! 2d8607d